client/extensions/wp-job-manager/app/controller.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import analytics from 'lib/analytics';
+import titlecase from 'to-title-case';
+import { getSiteFragment, sectionify } from 'lib/route';
+import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
+import { getSelectedSite } from 'state/ui/selectors';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import WPJobManager from './main';
+
+export function settings( context ) {
+	const siteId = getSiteFragment( context.path );
+	const site = getSelectedSite( context.store.getState() );
+	const { tab = '' } = context.params;
+
+	context.store.dispatch( setTitle( i18n.translate( 'WP Job Manager', { textOnly: true } ) ) );
```

Comment:
Can we use the [`DocumentHead`](https://github.com/Automattic/wp-calypso/tree/342c35adcb7ad07339b0754ac0ef7d30b1d8dde0/client/components/data/document-head) component inside the `WPJobManager` component instead?